src/Python-2.0.1.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license licenseId="Python-2.0.1" name="Python License 2.0.1" listVersionAdded="3.18">
+      <crossRefs>
+         <crossRef>https://www.python.org/download/releases/2.0.1/license/</crossRef>
+         <crossRef>https://docs.python.org/3/license.html</crossRef>
+         <crossRef>https://github.com/python/cpython/blob/main/LICENSE</crossRef>
+      </crossRefs>
+      <notes>
+         For Python release 1.6.1, apparently the CNRI-Python license part of the stack was replaced with CNRI-Python-GPL-Compatible
+      </notes>
+      <text>
+         <titleText>
+            <p>
+               PSF LICENSE AGREEMENT FOR PYTHON 3.10.6
+            </p>
+         </titleText>
+         <list>
+            <item>
+               <bullet>1.</bullet>
+               This LICENSE AGREEMENT is between the Python Software Foundation
+               ("PSF"), and the Individual or Organization ("Licensee")
+               accessing and otherwise using <alt match="this software \(\"Python\"\)|Python(( [0-9]+)\.([0-9]+)\.([0-9]+))?" name="software"> this software ("Python")</alt>
```

Comment:
Ah -- so this is failing because the quotation marks within the `match` attribute, even though they're escaped for purposes of the regex, the XML itself interprets them as ending the string. I'm digging to see if I can figure out an alternative approach here  :)